sql/table.cc

```diff
@@ -7774,3 +7777,17 @@ void TABLE::set_pos_in_table_list(TABLE_LIST *table_list) noexcept {
 }
 
 //////////////////////////////////////////////////////////////////////////
+
+const time_point get_time_now() { return std::chrono::system_clock::now(); }
+
+void TABLE_SHARE::set_last_access_time() noexcept {
+  this->last_accessed = get_time_now();
+}
+
+void TABLE::set_last_access_time() noexcept {
+  this->last_accessed = get_time_now();
+}
+
+bool should_be_evicted(time_point last_accessed, time_point cutpoint) {
```

Comment:
static

Reply:
The function is used in `table_cache.cc` and `table.cc`.